scripts/generate-sealed-secrets-pki.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -e
+
+VAULT_NAME=$1
+NAMESPACE=${2:-admin}
+
+openssl req -x509 -newkey rsa:4096 -keyout tls.key -out tls.crt -days 365 \
+  -subj "/C=US/ST=Oregon/L=Portland/O=Company Name/OU=Org/CN=www.example.com" -nodes
```

Comment:
does this need a proper cert as well?

Reply:
yes this should addressed and replaced with a format cert

Reply:
👍 needs doing